platform/viewer/src/connectedComponents/ConnectedStudyBrowser.js

```diff
@@ -28,13 +31,55 @@ const mapStateToProps = (state, ownProps) => {
   });
 
   return {
+    activeViewportIndex: state.viewports.activeViewportIndex,
     studies: studiesWithLoadingData,
   };
 };
 
+const mapDispatchToProps = dispatch => {
+  return {
+    setViewportSpecificData: (viewportIndex, data) => {
+      dispatch(setViewportSpecificData(viewportIndex, data));
+    },
+  };
+};
+
+const mergeProps = (propsFromState, propsFromDispatch, ownProps) => {
+  const findDisplaySet = (studies, studyInstanceUid, displaySetInstanceUid) => {
+    const study = studies.find(study => {
+      return study.studyInstanceUid === studyInstanceUid;
+    });
+
+    if (!study) {
+      return;
+    }
+
+    return study.displaySets.find(displaySet => {
+      return displaySet.displaySetInstanceUid === displaySetInstanceUid;
+    });
+  };
+
+  return {
+    ...propsFromState,
+    onThumbnailClick: data => {
```

Comment:
Is this just a previously unused prop?

Reply:
It was actually mergin a not used props `activeViewportIndex` that was only used inside mergeProps functions. We just need studies. 
Code is updated.